exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
+library(tidyverse)
+library(ggthemes)
+
+#Load malaria data Rstudio
+
+malaria_data <- read.table("https://raw.githubusercontent.com/OxfordIHTM/teaching_datasets/main/malaria.dat",header=TRUE)
+
+#View malaria data
+
+malaria_data
```

Comment:
It is good that you remember how to view the data. But in a script, this is not really needed. You can encode it here but I would suggest that you hash it. But as you progress, this line of code isn't really necessary. You implement this straight on console so you can see the data or on RStudio, click on the object name on the environment pane so you can view the data.